codecparsers/bitReader_unittest.cpp

```diff
@@ -73,4 +73,27 @@ BITREADER_TEST(NoOverflow)
     EXPECT_EQ(0u, reader.getRemainingBitsCount());
 }
 
+void checkBitread(BitReader& reader)
```

Comment:
It looks like this function is only relevant for checking a reader with a null/empty bitstream.  Perhaps rename it to checkBitreadEmpty or checkBitreadNull for clarity.